Conditional Statements and Data Types/code.js

```diff
@@ -0,0 +1,30 @@
+var readlineSync = require('readline-sync'); //module for interactively running to have a conversation with the user via a console
+
+let exit = true; //accept for now the user's desire to work in the program as TRUE
+while (exit === true) { //recircle programm while the user desire it
```

Comment:
There is a lot of redundancy in this assignment.
One of the principles of writing code is keep it simple.
The task is done correctly, but there is a lot of extra and unnecessary code to complete this task.